actions/commands.py

```diff
@@ -164,4 +164,34 @@ def my_tickets(request):
         text=message,
     )
 
-    return HttpResponse(status=200)
\ No newline at end of file
+    return HttpResponse(status=200)
+
+@csrf_exempt
+def set_description(request):
+    data = request.POST
+    channel_id = data.get("channel_id")
+    text = data.get("text")
+    args = ArgumentParser.parse_args(text)
+    user_id = data.get("user_id")
+    api_request = AuthorizedRequest(user_id=user_id)
+    ticket_desc = args.get("desc")
+    ticket_id = args.get("id")
+    message = f"Description for ticket {ticket_id} updated"
+
+    try:
+        response = api_request.patch(
+            url=f"http://127.0.0.1:8000/api/teams/13/tickets/{ticket_id}/",
+            data={"description": ticket_desc, "team_pk": 13}
```

Comment:
this isn't doing what you think it's doing. `"team_pk": 13" will have no effect since you're inserting it as part of the request data. (it would be sanitized by the serializers). It's fine for now, but the team needs to discuss how to handle teams properly, else we're just hardcoding everything and needing to manually change it every single time someone wants to merge it into master.